x-pack/test/idp-fixture/oidc/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM c2id/c2id-server-demo:12.18 AS c2id
 FROM openjdk:11.0.16-jre
 
+# TODO this is obviously horrible
+RUN apt-get update -qqy && apt-get install -qqy python3
```

Comment:
`python3` is to launch the "fake" server in `x-pack/test/idp-fixture/oidc/setup.sh` -- technically it's not necessary and we could start and stop the c2id-server but python was simpler, and I'm hesitant to sink more time into this, presently. Let me know if you feel this is too hacky and I can address in this PR, or a follow up though.